artifacts/deploy/webhook-configuration.yaml

```diff
@@ -75,6 +75,20 @@ webhooks:
     sideEffects: None
     admissionReviewVersions: [ "v1" ]
     timeoutSeconds: 3
+  - name: resourcebinding.karmada.io
```

Comment:
Subsequently, a new issue can be added to describe the task: synchronizing these changes with other installation methods.

Reply:
ok